plugin-viewport/src/test/kotlin/com/mapbox/maps/plugin/viewport/ViewportPluginImplTest.kt

```diff
@@ -288,10 +323,14 @@ class ViewportPluginImplTest {
 
   @Test
   fun testTransitionToDifferentStateWhileInState() {
-    val targetState = mockk<ViewportState>(relaxed = true)
-    val newState = mockk<ViewportState>(relaxed = true)
-    val transition = mockk<ViewportTransition>(relaxed = true)
-    val stateUpdatingCancelable = mockk<Cancelable>(relaxed = true)
+    val targetState = mockk<ViewportState>()
```

Comment:
nit:
maybe make `targetState`, `transition` as common field for test class and mock them in beforeEach method?